sklearn/cross_decomposition/_pls.py

```diff
@@ -505,6 +505,12 @@ class PLSRegression(_PLS):
     PLSRegression is also known as PLS2 or PLS1, depending on the number of
     targets.
 
+    For an example on usage, see
+    :ref:`sphx_glr_auto_examples_cross_decomposition_plot_compare_cross_decomposition.py`.
```

Comment:
this is more of a comparison rather than example usage, the comment should be informative of what the example is.

Reply:
Apologies for the delayed response. But doesn't that section show example usage of different `PLS*` algorithms?

Reply:
on second look at, it is more of a comparison rather than an example of how to use it (it is also mentioned as the title of the html page). Will change it accordingly